patches/dom-scroll-into-view+1.2.1.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/node_modules/dom-scroll-into-view/lib/dom-scroll-into-view.js b/node_modules/dom-scroll-into-view/lib/dom-scroll-into-view.js
```

Comment:
Could you remove this file?

Reply:
We can't. This patch is needed. Without the patch it won't scroll upward, only downward. You can check the CodeSandbox example for a proof.

Reply:
How des this file works? How it should be included when we move this code into Core? Should we propose a patch upstream instead?

Reply:
Also, is this useful for the "template editor" or just the "site editor"?

Reply:
This is useful for anything that's iframed.

> How des this file works? How it should be included when we move this code into Core?

We have a `postinstall` script that patches the package:

```json
"postinstall": "patch-package && node ./patches/patch-xcode.js",
```

`patch-package` applies the patch after `npm i`.

> Should we propose a patch upstream instead?
I will bring this up in the code's repo.